CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.12.0)
 
 project(ni_grpc_device_server
   LANGUAGES C CXX
-  VERSION 2.6.0)
+  VERSION 2.7.0)
```

Comment:
@maxxboehme i noticed that you bumped the version here. do you expect to make this fix in the main branch instead of releases/2.6.0?

Reply:
@yaqian256 , I had cherry picked this accidentally, I will revert it in this PR.

I initially only changed `functions.py` but ran into errors on the pipeline saying some of the github actions used were outdated.  I ended up cherry picking relevant changes from main but looks like I included the cmake changes incorrectly.